src/backend/services/ExtractZipService.ts

```diff
@@ -0,0 +1,419 @@
+import { EventEmitter } from 'node:events'
+import { Readable } from 'node:stream'
+import { open, ZipFile, Entry } from 'yauzl'
+import { mkdirSync, createWriteStream, rmSync, existsSync } from 'graceful-fs'
+import { captureException } from '@sentry/electron'
+import { join } from 'path'
+
+export interface ExtractZipProgressResponse {
+  /** Percentage of extraction progress. */
+  progressPercentage: number
+  /** Speed of extraction in bytes per second. */
+  speedInMbPerSec: number
```

Comment:
wait is this one bytes per second or megabytes per second?

Reply:
Oh dam, was a typo. Cheers for noticing 🍻

Reply:
Updated